sdk/keyvault/azure-keyvault-keys/tests/recordings/test_crypto_client.test_decrypt_valid_arguments.yaml

```
@@ -0,0 +1,102 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '0'
+      Content-Type:
+      - application/json
+      User-Agent:
+      - azsdk-python-keyvault-keys/4.3.2 Python/3.5.3 (Windows-10-10.0.19041-SP0)
+    method: POST
+    uri: https://vaultname.vault.azure.netkeys/livekvtestrsa-keyd502138b/create?api-version=7.2-preview
```

Looks like a recording processor isn't replacing the URL correctly. This should be 
```suggestion
    uri: https://vaultname.vault.azure.net/keys/livekvtestrsa-keyd502138b/create?api-version=7.2-preview
```
I saw that happen in one of your other PRs but it was fixed after another recording. Is this branch missing changes?

---

Yeah, this happened earlier during a recording, even though I hadn't made any changes that would have explained it. I'll try recording again -- I also see that some tests recorded a 7.1 API version for some reason

---

As expected, the API version issue was due to a misconfiguration on my part. 7.1 was used on async tests only because I hadn't updated the default API version for the async client base class (or moved over generated multiapi aio classes). Learning experiences abound